Description: make linuxpba statically built
 If this is going into an initramfs or similar, we need it to be
 self-sufficient.

--- a/Makefile.am
+++ b/Makefile.am
@@ -41,6 +41,7 @@ linuxpba_SOURCES = LinuxPBA/LinuxPBA.cpp
 	LinuxPBA/GetPassPhrase.h LinuxPBA/UnlockSEDs.h \
 	$(SEDUTIL_LINUX_CODE) \
 	$(SEDUTIL_COMMON_CODE)
+linuxpba_LDFLAGS = -static $(AM_LDFLAGS)
 EXTRA_DIST = linux/GitVersion.sh linux/PSIDRevert_LINUX.txt linux/TestSuite.sh README.md docs/sedutil-cli.8
 man_MANS = docs/sedutil-cli.8
 linux/Version.h:
